core/ui/common/src/main/java/io/filmtime/core/ui/common/componnents/LoadingCastSectionRow.kt

```diff
@@ -0,0 +1,51 @@
+package io.filmtime.core.ui.common.componnents
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import io.filmtime.core.ui.common.graphics.shimmerBrush
+
+@Composable
+fun LoadingCastSectionRow(
+  numberOfSections: Int,
+) {
+  LazyRow() {
+    items(numberOfSections) {
+      Column(
+        modifier = Modifier
+          .fillMaxWidth().wrapContentHeight()
+          .padding(horizontal = 6.dp),
```

Comment:
would you please break down the modifiers into separate lines like this?

```suggestion
        modifier = Modifier
          .fillMaxWidth()
          .wrapContentHeight()
          .padding(horizontal = 6.dp),
```